### PR TITLE
loading ScanNet scans in habitat-sim

### PR DESCRIPTION
**Pull request by srama2512**
## Motivation and Context

This PR provides instructions to load ScanNet scans in habitat-sim. This addresses issue #1142. 

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by dhruvbatra** [comment]
Need to run pre-commit. But other than that looks good! Thanks!

**Review by dhruvbatra — APPROVED** [review]

File: DATASETS.md [hunk 1]
```diff
@@ -64,3 +64,15 @@ Note: To obtain the best rendering results, use the `<path to replica>/<scene_na
 ## ReplicaCAD
 
 Details and download instructions: [https://aihabitat.org/datasets/replica_cad/](https://aihabitat.org/datasets/replica_cad/).
+
+## ScanNet
+
+The official ScanNet data can be downloaded here: [http://www.scan-net.org/](http://www.scan-net.org/). To use ScanNet scans with habitat-sim, the `scene_*.ply` files need to be converted to `*.glb` format using [assimp](https://github.com/assimp/assimp).
```
Comment (aclegg3) [inline DATASETS.md:70]:
```suggestion
The official ScanNet data can be downloaded here: [http://www.scan-net.org/](http://www.scan-net.org/). To use ScanNet scans with habitat-sim, the `scene_*.ply` files need to be converted to glTF format (`*.glb`). For example, using [assimp](https://github.com/assimp/assimp):
```

File: DATASETS.md [hunk 2]
```diff
@@ -64,3 +64,15 @@ Note: To obtain the best rendering results, use the `<path to replica>/<scene_na
...
+
+```
+assimp export <PLY FILE> <GLB PATH> 
+```
+
+The exported `*.glb` files can directly be used with habitat-sim versions >= 2.0.
+
+Note: Depending on the radius and height of the robot, certain scans may have no navigable locations on the navmesh (~200). These scenes can be filtered out by checking if `sim.pathfinder.is_loaded` is False.
```
Comment (aclegg3) [inline DATASETS.md:78]:
```suggestion
Note: Depending on the configured radius and height of the agent, certain scans may have no navigable locations on the navmesh (~200). These scenes can be filtered out by checking if `sim.pathfinder.is_loaded` is False.
```

**Review by aclegg3 — APPROVED** [review]
Thanks for adding this! A couple of minor pedantic suggestions. 🙂

**Comment by srama2512** [comment]
Thanks for the suggestions @aclegg3 ! I've updated the PR with the suggested changes.

**Comment by srama2512** [comment]
@aclegg3  Not sure why circleci is failing, especially considering that this is only a documentation change. Otherwise, this looks ready to me.

**Review by Skylion007 — APPROVED** [review]

**Pull request merged**